cds/modules/theme/static/js/cds/module.js

```diff
@@ -576,3 +576,24 @@ app.directive('popover', function() {
       element.find('a[rel=popover]').popover();
   };
 });
+
+/**
+ * filter for shating on different social platforms
+ * more details about how we build the URLs you can find on: https://github.com/bradvin/social-share-urls
+ */
+app.filter('assembleShareURL', ['$window', function($window) {
+  return function(record, platform) {
+    if (!record) { return; }
+
+    var currentPageAddress = $window.location.href;
```

Comment:
you should encode arguments value with `encodeURIComponent` if I am not mistaken.
`encodeURIComponent(currentPageAddress)` or `encodeURIComponent(record.medata.description)`
because if inside you have a `&` or `?` for example, it will split your arguments.

Please do a test with a title and description with such characters and also with a language with accents in the text, spanish, italian, french for example.